odbc/src/api/definitions.rs

```diff
@@ -555,12 +556,25 @@ pub const SQL_ROW_NUMBER_UNKNOWN: isize = -2;
 pub const SQL_CB_NULL: u16 = 0x0000;
 pub const SQL_U16_ZERO: u16 = 0x0000;
 pub const SQL_CL_START: u16 = 0x0001;
-pub const SQL_U32_ZERO: u32 = 0;
+pub const SQL_U32_ZERO: u32 = 0x0;
+pub const SQL_U32_ALL: u32 = 0xffffffff;
 pub const SQL_OIC_CORE: u32 = 0x00000001;
 pub const SQL_SC_SQL92_ENTRY: u32 = 0x00000001;
 pub const SQL_INFO_Y: &str = "Y";
 pub const SQL_GB_GROUP_BY_CONTAINS_SELECT: u16 = 0x0002;
 pub const SQL_CB_PRESERVE: u16 = 2;
+pub const SQL_CA1_NEXT: u32 = 0x00000001;
+pub const SQL_CA2_READ_ONLY_CONCURRENCY: u32 = 0x00000001;
+pub const SQL_CA2_MAX_ROWS_SELECT: u32 = 0x00000080;
+pub const SQL_CA2_CRC_EXACT: u32 = 0x00001000;
+pub const MONGO_CA2_SUPPORT: u32 =
+    SQL_CA2_CRC_EXACT | SQL_CA2_MAX_ROWS_SELECT | SQL_CA2_READ_ONLY_CONCURRENCY;
```

Comment:
Do we actually implement the logic for honouring the max rows setting?

Reply:
Actually, that's a good point. Let me remove that.